services/configuration/src/schema.rs

```diff
@@ -54,13 +55,51 @@ lazy_static! {
     ).into_bytes();
 }
 
-/// A functional equivalent to `Option<Vote>` used to store votes in the service schema.
+/// A enum used to represent different kinds of vote, `Vote` and `VoteAgainst` transactions.
+#[derive(Clone, Debug, PartialEq, Serialize, Deserialize)]
+#[serde(tag = "vote_for")]
```

Comment:
I'd also add [`#[serde(rename_all = "lowercase")]` attribute](https://serde.rs/container-attrs.html#serderenameall--); lower-case discriminators are more common in JSON.